### ISSUE TITLE
Numeric columns: initial stats query

### ISSUE THREAD

**Issue by dandanlen** [diffix/explorer#7]
We need an initial query for numeric columns to inform the shape of further exploration. 

We need to retrieve measures for the `min` and `max`, total `count`, might as well get the `avg` and `stddev` too. 

Something like:

```sql
select min({column}), max({column}), count(*), avg({column}), stddev({column})
from {table}
group by {column}
```

Using this, we can for example estimate the range and distribution of the data to decide which bucket sizes to query for.

**Comment by sebastian** [comment]
Please note that `stddev` has an un-optimized implementation in Aircloak Insights. It will be unbearably slow on large data sets. Depending on the installation it might cause the query to be cancelled with an "out of memory warning", cause the system to crash and furthermore put a significant strain of both bandwidth, and the db and cloak.

If you can get the results you need by other means that would be good.

**Comment by dandanlen** [comment]
Closed #32
